pysat/__init__.py

```diff
@@ -60,16 +60,16 @@
     os.mkdir(pysat_dir)
     print('Created .pysat directory in user home directory to store settings.')
     # create file with default data directory
-    if not (os.environ.get('TRAVIS') == 'true'):
-        data_dir = os.path.join(home_dir, 'pysatData')
+    if (os.environ.get('TRAVIS') == 'true'):
+        data_dir = '/home/travis/build/pysatData'
     else:
-        data_dir = '/home/travis/build/rstoneback/pysatData'
+        data_dir = ''
```

Comment:
Won’t this just dump all the data to the current directory?

Reply:
No, I put a check in the files class that ensures data_dir is not ''. Line 152 in _files.py.